### DISCUSSION TITLE
File transfer window - Issues (size column too big / american date/time format)

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#5161] [Feature Request]
@rustdesk

### Bug Description

In File tarnsfer window the Size colum width is always bigger then what is required and cannot reduce less then what is reuired.

Then the file name width cannot be increase why Size column is nor resizable elss

The date/time format is in american format (yyyy/dd/mm).

I hope that in a enxt rerlase the date/time format can set automtically absed on GUI language.

### How to Reproduce

Use file transfer

### Expected Behavior

Size column smaller
File name can incraese
Date/time format set in coording on GUI language selected

### Operating system(s) on local side and remote side

Windows 10 22H2 64bit italian

### RustDesk Version(s) on local side and remote side

1.2.2 27.07.2023

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/1262554/60282e86-700d-4826-aee9-2cad19c389b0)

**Comment by bovirus** [comment]
@rustdesk 

Why move to "discussion"? Form me are bugs.
Size column width too big is a bug (cannot be resizable less then a value that is too big for the real requirement).
The file date/time format not follow the specfication for interface.
If I select Italian as language my expectation as italian user is to have Italian date/time format (not american date/time format).

**Comment by ghost** [comment]
Agree with OP about US date / time format. There are 100+ other countries in the world - why default to American format? You'll be using inches and feet next....!

**Comment by bovirus** [comment]
@RichardBurt65
Plase be serious. Is not necesary inchees and feet (are not date/time format). 
When you select the language for GUI you can set automically the relative date/time format. As Windows do.

**Reply by ghost** [reply]
My reply to was to the devs, not you. I agree with you completely. 

What they need is the option of Eng (International) and Eng (USA)

**Comment by bovirus** [comment]
@RichardBurt65
Your first answer (inchees and feet) seems likes a joke.
Please don't think only about english poeple.
I'm italian and if i select Italian language for GUI my expectation is to have italian date/time format.
And I believe that will be the same for other languages.

**Comment by rustdesk** [comment]
is this en date format? not iso international format?

**Reply by rustdesk** [reply]
of course, locale date format is better and more friendly for users.
